### PR TITLE
Add deploy from bytes option

### PR DESCRIPTION
**Pull request by PrathamSoni**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by thomasht86** [comment]
Thanks for your contribution once again @PrathamSoni. 
Only a minor change request from me, before we can merge this. 👍

**Comment by PrathamSoni** [comment]
hey @thomasht86 don't see the coment? maybe you need to publish it?

**Comment by thomasht86** [comment]
Hi @PrathamSoni, sorry for delayed response. We are in the process of migrating our CI pipelines, and we have not configured Github actions triggered by fork PRs to have access to secrets, which means that all runs triggered by fork PRs currently fail. We are looking to address this soon.

**Comment by thomasht86** [comment]
Temporarily closing until we have resolved how we to this to avoid reruns.

**Comment by PrathamSoni** [comment]
@thomasht86 is it ok to reopen :D

**Comment by thomasht86** [comment]
Yes! Sorry, I didn't remember.

File: vespa/deployment.py [hunk 1]
```diff
@@ -207,6 +207,33 @@ def deploy(
             docker_timeout=max_wait_docker,
             debug=debug,
         )
+    
+    def deploy_from_bytes(
```
[the remark this answers is not in the document]
Comment (thomasht86) [reply vespa/deployment.py:211]: This was the comment I was referring to @PrathamSoni

**Pull request closed without merging**